docs_src/microservices/application/AppFunctionContextAPI.md

```diff
@@ -196,13 +197,18 @@ This API returns that timestamp for when the secrets in the SecretStore where la
 
 ## Miscellaneous
 
+### Clone()
+`Clone() AppFunctionContext`
+
+This method returns a copy of the context that can be mutated independently where appropriate.  This is can be useful when running operations that take AppFunctionContext in parallel.
```

Comment:
```suggestion
This method returns a copy of the context that can be mutated independently where appropriate.  This can be useful when running operations that take AppFunctionContext in parallel.
```